src/react-components/banner/Banner.js

```diff
@@ -0,0 +1,321 @@
+              id="banner.success-message"
+            />
+          </p>
+          <Button
+            preset="primary"
+            onClick={() => {
+              setIsExpanded(false);
+            }}
+          >
+            <FormattedMessage defaultMessage="Got it" id="banner.success-cta" />
+          </Button>
+        </div>
+      </div>
+    );
+  };
+
+  /**
+   * Error Message
+   * @returns JSX
+   */
+  const Error = () => {
+    return (
+      <div className={styles.message_wrapper}>
+        <div>
+          <img src={warning_icon} className={styles.error_icon} />
+        </div>
+        <div>
+          <h3 className={styles.message_title}>
+            <FormattedMessage defaultMessage="We ran into a problem" id="banner.error-title" />
+          </h3>
+
+          <p className={styles.message_body}>
+            <FormattedMessage
+              defaultMessage="{message}"
+              id="banner.error-body"
+              values={{
+                message: (
+                  <>
+                    Sorry, we were unable to add you to the mailing list, please try again later. If the problem
+                    persists please reach out on our{" "}
+                    <a href="https://discord.com/invite/dFJncWwHun" rel="noopener noreferrer" target="_blank">
+                      Discord
+                    </a>.
+                  </>
+                )
+              }}
+            />
+          </p>
+          <Button
+            preset="primary"
+            onClick={() => {
+              setIsExpanded(false);
+            }}
+          >
+            <FormattedMessage defaultMessage="Got it" id="banner.error-cta" />
+          </Button>
+        </div>
+      </div>
+    );
+  };
+
+  return (
+    <div className={styles.banner_wrapper}>
+      <div className={styles.banner_container}>
+        {/* BRANDING  */}
+        <div className={styles.branding_container}>
+          <h2>
+            <FormattedMessage defaultMessage="Join the next evolution of Hubs!" id="banner.title" />
+          </h2>
+          <p>
+            <FormattedMessage defaultMessage="Be the first to get a sneak peek!" id="banner.subtitle" />
+          </p>
+        </div>
+
+        <Button
+          preset="primary"
+          onClick={() => {
+            setIsExpanded(true);
+          }}
+        >
+          <FormattedMessage defaultMessage="Find out More" id="banner.expand-cta" />
+          <span className={styles.down_arrow}>
+            <ArrowIcon />
+          </span>
+        </Button>
+      </div>
+
+      {/* EXPAND CONTAINER  */}
+      {isExpanded ? (
+        <div className={styles.expand_wrapper}>
+          <div className={styles.expand_container}>
+            <div className={styles.expand_header}>
+              <h2 className={styles.expand_title}>
+                <FormattedMessage defaultMessage="Join the next evolution of Hubs!" id="banner.expand_title" />
```

Comment:
Use a hyphen to keep these ids consistent.
```suggestion
                <FormattedMessage defaultMessage="Join the next evolution of Hubs!" id="banner.expand-title" />
```